pkg/arvo/sys/vane/ames.hoon

```diff
@@ -3976,9 +3993,11 @@
               ::
               =?  peer-core  !=(~ next-wake.state)
                 (pu-emit %b %rest (need next-wake.state))
-              ::  set new timer if non-null
+              ::  set new timer if non-null and not at max-backoff
               ::
               =?  peer-core  ?=(^ new-wake)
+                ?:  =(~m2 rto.metrics.state)
```

Comment:
Worth noting here (and above) that the `~m2` literal is important, we don't want to actually use `max-backoff` and accidentally consolidate the `:ping` app flow.

Reply:
Added a comment in a3e75959b13e4f2b0c61ca909902e9f1e05b1d76